src/main/resources/db/migration/common/V1_387__audio_linking_task.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size,
```

Comment:
This should probably be disabled whilst UAT is going on, you able to update it? I'll create a ticket to make sure we re-enable once that is finished.

Reply:
Updated